### PR TITLE
fix: pin the version of jaxlib (checks are failing)

### PR DESCRIPTION
**Pull request by OmaymaMahjoub**
## What?
Pin the version of jaxlib
## Why?
Checks are failing because of this error
`jaxlib version 0.3.20 is newer than and incompatible with jax version 0.3.15. Please update your jax and/or jaxlib packages.`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
     # TODO Unpin JAX, once new rlax and acme version is released.
     # To fix cannot import name 'tree_multimap' from 'jax.tree_util'.
     "jax<=0.3.15",
-    "jaxlib",
+    "jaxlib<=0.3.19",
```
Comment (KaleabTessera) [inline setup.py:44]: Thanks @OmaymaMahjoub !  We should pin jax and jaxlib to the same versions.

**Review by KaleabTessera — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (OmaymaMahjoub) [reply setup.py:44]: Okay! Should we keep jax to 0.3.15 or change it to 0.3.17 since it works fine in that version too?

Comment (KaleabTessera) [reply setup.py]: I updated it :) . Please let me know if you are happy?

Comment (OmaymaMahjoub) [reply setup.py]: Thank you! I think the latest jax is 0.3.18 and it didn't work in our case bringing some errors so probably we can just stay with 0.3.17 so let's wait to see if the checks are fine else we can re edit it

Comment (OmaymaMahjoub) [reply setup.py]: It's working fine 🙂 thank you 🙌

**Review by KaleabTessera — APPROVED** [review]
Thanks @OmaymaMahjoub ! 

@RuanJohn Can we please run an exp or two to sanity check upgrading jax doesn't have any weird side effects? :)

**Comment by RuanJohn** [comment]
> Thanks @OmaymaMahjoub !
> 
> @RuanJohn Can we please run an exp or two to sanity check upgrading jax doesn't have any weird side effects? :)

On it :+1:

**Review by RuanJohn — APPROVED** [review]
Looks good and the benchmarks show no noticeable performance decreases.

**Pull request merged**